runtimes/dart-2.12/pubspec.yaml

```diff
@@ -7,11 +7,6 @@ environment:
   sdk: '>=2.12.0 <3.0.0'
 
 dependencies:
-  path: ^1.8.0
-  alfred: ^0.1.4+5
   function_types:
-    path: ./function_types/
+    git: https://github.com/lohanidamodar/function_types
```

Comment:
Is this required for the runtime ? I believe we were going to wait with the types and add them in our SDK later.